linode/manager
# change: [M3-7205] - Dark theme Radio & Toggle improvements

## Description 📝
This PR improves the styling of the Radio and Toggle components in dark mode (only). It aims to handle default and disabled states styling when the dark mode is turned on.

**It does not modify the light theme styling**. 

## Changes  🔄
- Improve styling for both components (dark mode only)
- Improve stories for both components
- Improve tooltip content in Migration panel (unified migration)

## Preview 📷
(note: the "Before" screenshot for the Resize modal is for a Linode that is powered off)

| Before  | After   |
| ------- | ------- |
| ![Screenshot 2024-01-02 at 14 37 50](https://github.com/linode/manager/assets/130582365/6d54b04f-da39-4377-9da8-b81f771b369a) | ![Screenshot 2024-01-02 at 14 33 49](https://github.com/linode/manager/assets/130582365/0e074ae4-f62b-4608-bc76-401fa47e0432) |
| ![Screenshot 2024-01-02 at 14 37 12](https://github.com/linode/manager/assets/130582365/0ece8f77-25ee-4123-82f6-d3e1572d2fab) | ![Screenshot 2024-01-02 at 14 35 33](https://github.com/linode/manager/assets/130582365/b0962311-206e-451b-a891-ae4621e63424) |

## How to test 🧪

### Verification steps 
- pull and run code locally
- run `yarn storybook` as well
- Verify `Toggle` and `Radio` component stories and the styling with the dark mode
- Verify styling of Radios and Toggles in CM (ex: user permissions for toggles and resize modal)

## As an Author I have considered 🤔

*Check all that apply*
